lib/node_modules/@stdlib/math/base/special/logit/README.md

```diff
@@ -0,0 +1,69 @@
+# logit
+
+> Compute the [logit][logit] of a proportion.
```

Comment:
Perhaps include a quick intro, explaining what the logit function is; e.g., the inverse of the logistic function, as well as its functional (equation) definition.